rocketc/rocketc.py

```diff
@@ -628,7 +628,7 @@ def create_group(self, data, suffix=""):
     def _create_team_group_name(team, group_name, course):
 
         team_name, team_topic = generate_team_variables(team)
-        return "{}__{}__{}__{}".format(group_name, team_name, team_topic, course)
+        return "{}({}/{})__{}".format(group_name, team_topic, team_name, course)
```

Comment:
Could you please add an example comment of the formating that you are doing here? @andrey-canon

Reply:
Please read the jira ticket added in the description

Reply:
It's not for me, it's for who will read the code in the future. @andrey-canon

Reply:
It's not for me, it's for who will read the code in the future. @andrey-canon

Reply:
done